src/main/java/frc/robot/subsystems/gpm/Intake.java

```diff
@@ -125,9 +139,56 @@ public boolean hasNote() {
     public void periodic() {
         publish();
 
-        // set the motor powers to be the value appropriate for this mode
-        motor.set(mode.power);
-        centeringMotor.set(mode.centeringPower);
+        // increment the number of clicks that we have been in this mode
+        counter++;
+
+        switch (mode) {
+            case DISABLED:
+            // don't have to do anything
```

Comment:
This switch statement is unnecessary. If you want to reverse the intake after the note gets stuck, then use a Debouncer. You can specify a time in seconds (instead of frames) that the sensor has to be activated to switch to reverse mode, and then maybe have another Debouncer or a Timer (instead of a counter) to set it to disabled mode after the note leaves. Also, I think the driver should have control over reversing the intake so we don't randomly outtake a note in front of a defender.

Reply:
I think that the switch statement works,
however I do agree that maybe we should have a "jam" message instead of automatically outtaking the note.

Reply:
I changed the switch statement to something better, but I'm not sure what to do instead of outtaking the note. Printing probably won't work because the driver won't see that. Do you have any ideas?

Reply:
Iliana figured out how to make the controller shake so we can let the driver know that way